docs/src/componentDocs/Drawer/examples/CustomizableDrawerExample.tsx

```diff
@@ -39,10 +36,7 @@ export const CustomizableDrawerExample = (): JSX.Element => (
               name: 'bell',
               direction: 'auto',
             }}
-            activeItemBackgroundShape={'round'}
-            InfoListItemProps={{
-              iconAlign: 'center',
-            }}>
+            activeItemBackgroundShape={'round'}>
             <DrawerNavItem itemID={'item3'} title={'Web'} />
```

Comment:
Can you select this item default so we can see `activeItemBackgroundShape` prop working?